x/examples/outline-connectivity/main.go

```diff
@@ -150,6 +184,13 @@ func main() {
 			if err != nil {
 				log.Fatalf("Failed to output JSON: %v", err)
 			}
+			// Send error report to collector if specified
+			if !success && *collectorFlag != "" {
```

Comment:
Let's add flags for success and failure sampling rates, like NEL: https://www.w3.org/TR/network-error-logging/#sampling-rates

Perhaps `-report-success-rate` and `-report-failure-rate`, so they all start with `-report-`.

Perhaps set defaults to 0.1 and 1.0?

Reply:
addressed on new PR